quirks/password-rules.json

```diff
@@ -60,16 +60,16 @@
         "password-rules": "minlength: 8; maxlength: 8; required: lower; required: upper; required: digit;"
     },
     "bankofamerica.com": {
-        "password-rules": "minlength: 8; maxlength: 20; required: upper; required: lower; required: digit; max-consecutive: 3; allowed: [-@#*()+={}/?~;,._];"
+        "password-rules": "minlength: 8; maxlength: 20; max-consecutive: 3; required: lower; required: upper; required: digit; allowed: [-@#*()+={}/?~;,._];"
     },
     "battle.net": {
-        "password-rules": "required: upper,lower; allowed: digit,special; minlength: 8; maxlength: 16;"
+        "password-rules": "minlength: 8; maxlength: 16; required: lower, upper; allowed: digit, special;"
     },
     "benjerry.com": {
-        "password-rules": "required: upper; required: upper; required: digit; required: digit; required: special; required: special; allowed: lower;"
+        "password-rules": "required: upper; required: digit; required: special; allowed: lower;"
```

Comment:
This isn't semantically equivalent to the rule as it was previously.

The previous rule was:

```
required: upper; required: upper; required: digit; required: digit; required: special; required: special; allowed: lower;
```

This requires the password to have:

* At least two uppercase letters
* At least two digits
* At least two special characters

However, the rule as you've written it is more relaxed, requiring:

* At least one uppercase letter
* At least one digit
* At least one special character

If you look at the PR that added this rule (#304) you'll notice that the relaxed requirements wouldn't be appropriate for this site.

Reply:
🤦🏻‍♂️ Oops, looks like I misunderstood the following line from the [Apple Developer document](https://developer.apple.com/documentation/security/password_autofill/customizing_password_autofill_rules) then: 
> You can combine these keywords to form your rule. _Duplicate property values_, empty character classes, and properties without a value _are ignored_.

Thanks for catching that, I'll fix it.

Reply:
Yeah, I agree it can be confusing; the `required` property is the exception to that.